pkg/apis/componentconfig/types.go

```diff
@@ -339,3 +343,24 @@ type LeaderElectionConfiguration struct {
 	// leader election is enabled.
 	RetryPeriod unversioned.Duration `json:"retryPeriod"`
 }
+
+// AddFlags binds the common LeaderElectionCLIConfig flags to a flagset
+func (l *LeaderElectionConfiguration) AddFlags(fs *pflag.FlagSet) {
```

Comment:
This is a types.go file. There is no presidence for putting functions in a types.go or defining function on API types.